ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -1940,6 +1940,35 @@ def test_tensorflow_rsqrt(
         x=x[0],
     )
 
+#segment_sum
+@handle_frontend_test(
+    fn_tree="tensorflow.math.segment_sum",
+    data=helpers.array_values(dtype=ivy.int32, shape=(5, 6), min_value=1, max_value=9),
+    segment_ids=helpers.array_values(
+        dtype=ivy.int32, shape=(5,), min_value=0, max_value=4
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_segment_sum(
```

Comment:
Hello! The test is failing for all backends. You can see the stacktrace and error in the details section of the intelligent-tests (1) 

Let me know if you have any questions!